.github/workflows/docker-build-push.yml

```
@@ -4,7 +4,7 @@ name: Build, Render, and Push
 # events only for the master branch
 on:
   push:
-    branches: [ master ]
+    branches: [ staging, master ]
```

Just for testing, do you want to try 
```suggestion
    branches: [ staging, master, cansavvy/gha-updates-pr-process ]
```
for a round or two?